docs/src/userguide/plotting_a_cube.rst

```diff
@@ -104,7 +106,7 @@ Some of the formats which are supported by **plt.savefig**:
 ======  ======  ======================================================================
 Format  Type    Description
 ======  ======  ======================================================================
-EPS     Vector  Encapsulated PostScript
+EPS     Vector  Encapsulated PostScript B
```

Comment:
@ESadek-MO What does the extra `B` mean? Or is it a typo?

Reply:
A typo! Seems I snuck a couple in accidentally, will clean up :)